packages/crayons-extended/platform-ui/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@freshworks/platform-ui",
+  "version": "1.0.0",
+  "description": "Extended crayons components",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.js",
+  "es2015": "dist/esm/index.mjs",
+  "es2017": "dist/esm/index.mjs",
+  "types": "dist/types/components.d.ts",
+  "collection": "dist/collection/collection-manifest.json",
+  "collection:main": "dist/collection/index.js",
+  "unpkg": "dist/platform-ui/platform-ui.esm.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ionic-team/stencil-component-starter.git"
+  },
+  "files": [
+    "dist/",
+    "loader/",
+    "react/"
+  ],
+  "scripts": {
+    "add:react": "npm run clean:react && npx mkdirp react",
+    "add:css": "npm run clean:css && npx mkdirp css",
```

Comment:
add:css is the first script followed by add:react

Reply:
Modified the order